libraries/aws_ebs_snapshots.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsEbsSnapshots < AwsResourceBase
+  name 'aws_ebs_snapshots'
+  desc 'Verifies settings for a collection of AWS EBS Snapshots'
+  example '
+    describe aws_ebs_snapshots do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:snapshot_ids, field: :snapshot_id)
+             .register_column(:owner_ids,    field: :owner_id)
+             .register_column(:encrypted,    field: :encrypted)
+             .register_column(:tags,         field: :tags)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    snapshot_rows = []
+    pagination_options = {}
+    loop do
+      catch_aws_errors do
+        @api_response = @aws.compute_client.describe_snapshots(pagination_options)
+      end
+      return [] if !@api_response || @api_response.empty?
+
+      @api_response.snapshots.map do |snapshot|
+        snapshot_rows += [{ snapshot_id: snapshot.snapshot_id,
+                            owner_id:    snapshot.owner_id,
+                            encrypted:   snapshot.encrypted,
+                            tags:        snapshot.tags }]
+      end
+      break unless @api_response.next_token
+      pagination_options = { next_token: @api_response.next_token }
+    end
+    @table = snapshot_rows
```

Comment:
This is redundant since you store it again here https://github.com/inspec/inspec-aws/pull/485/files#diff-c6371d4fda843df9ab8ab530d2245ce52e997772442cfe33887e1d51ed5f142dR26, either one of it is fine.

Reply:
Right, another thing I didn't catch on review of my new team mate's code. Thanks :)

Reply:
Removed the assignment from the call, left it in the method.